src/test/java/e2e/SingleModuleTest.java

```diff
@@ -100,4 +100,18 @@ private ObjectId head(Git git) throws IOException {
         return git.getRepository().getRef("HEAD").getObjectId();
     }
 
+    @Test
+    public void originTagsNotConsultedWithoutPull() throws Exception {
```

Comment:
Demonstrates that, when `pullTags` is disabled, the origin tag doesn't affect the release number calculation. This isn't actually how the switch is supposed to be used (it doesn't make sense to have divergent release calculations!) but it does demonstrate functionality.